integration/nwo/token/generators/generators.go

```diff
@@ -7,11 +7,24 @@ SPDX-License-Identifier: Apache-2.0
 package generators
 
 import (
+	api2 "github.com/hyperledger-labs/fabric-smart-client/integration/nwo/api"
+	"github.com/hyperledger-labs/fabric-smart-client/integration/nwo/common"
 	"github.com/hyperledger-labs/fabric-smart-client/integration/nwo/fsc/node"
-
 	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/token/topology"
+	"github.com/onsi/gomega/gexec"
 )
 
```

Comment:
shall we add comments here to describe the methods?

Reply:
okay